### PR TITLE
migrate docs and fix links to plugin

### PR DESCRIPTION
**Pull request by Yunir**
cause: wiki-pages migration to jenkins.io

### CODE REVIEW

**Comment by Yunir** [comment]
There is one more wiki-page URL in **README.md:173:** but I can not change it to jenkins.io because it has not migrated yet.

**Comment by Yunir** [comment]
@halkeye, if u say me how can I migrate [Starting+and+Accessing+Jenkins](https://wiki.jenkins-ci.org/display/JENKINS/Starting+and+Accessing+Jenkins) to jenkins.io, then I can set the status WIP for that PR and finish it after anyone (maybe I) migrate this wiki-page.

**Comment by MarkEWaite** [comment]
See comments in the gitter chat as a proposal.

File: pom.xml [hunk 1]
```diff
@@ -11,12 +11,12 @@
     <version>2.2.0-SNAPSHOT</version>
     <packaging>hpi</packaging>
 
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/GitHub+PR+Coverage+Status+Plugin</url>
+    <url>https://plugins.jenkins.io/github-pr-coverage-status</url>
```
Comment (timja) [inline pom.xml]:
```suggestion
    <url>https://github.com/jenkinsci/github-pr-coverage-status-plugin</url>
```

**Comment by Yunir** [comment]
It is better now create an issue to transport content from [wiki - Starting+and+Accessing+Jenkins](https://wiki.jenkins.io/display/JENKINS/Starting+and+Accessing+Jenkins) to [jenkins.io - installing](https://jenkins.io/doc/book/installing/) separately.
For now I just set link to https://jenkins.io/doc/book/installing/ and finish this PR.

**Comment by Yunir** [comment]
@MarkEWaite I created PR for migration of that wiki-page: https://github.com/jenkins-infra/jenkins.io/pull/2869

**Pull request merged**